stingray/powerspectrum.py

```diff
@@ -469,10 +478,13 @@ def from_events(*args, **kwargs):
             the real part
         """
```

Comment:
Looks like there's a number of options in the docstring that are not actually options in the method?

Do we no longer care about these options? I feel like there were science cases where people wanted the full spectrum with negative frequencies and the complex powers?

Reply:
Yes, it was just a bad copy/paste. They make perfect sense in crossspectrum, not here.